prow/jobs/kyma/tests/end-to-end/backup-restore-test/backup-restore-test.yaml

```diff
@@ -0,0 +1,49 @@
+job_template: &job_template
+  skip_report: false
+  run_if_changed: "^tests/end-to-end/backup-restore-test/"
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-project/kyma/tests/end-to-end/backup-restore-test"
+        resources:
+          requests:
+            memory: 1.5Gi
+            cpu: 0.8
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-sa-gcr-push: "true"
+  preset-docker-push-repository: "true"
+
+presubmits: # runs on PRs
```

Comment:
what about release job? if we plan to have it for 0.8 you should add definition

Reply:
Hi Adam I had to remove the test for the release jos as it keep failing:
```

--- FAIL: TestEndToEndBackupRstoreTestReleases (0.00s)
--
308 | --- FAIL: TestEndToEndBackupRstoreTestReleases/release-0.6 (0.00s)
309 | require.go:919:
310 | Error Trace:	end_to_end_backup_restore_test_test.go:64
311 | Error: Expected value not to be nil.
312 | Test: TestEndToEndBackupRstoreTestReleases/release-0.6
313 | --- FAIL: TestEndToEndBackupRstoreTestReleases/release-0.7 (0.00s)
314 | require.go:919:
315 | Error Trace:	end_to_end_backup_restore_test_test.go:64
316 | Error: Expected value not to be nil.
317 | Test: TestEndToEndBackupRstoreTestReleases/release-0.7
318 | FAIL
319 | FAIL	github.com/kyma-project/test-infra/development/tools/jobs/kyma	0.603s
```

Reply:
I think because the test is not part of the release jet.

Reply:
Can the PR still merged?

Reply:
yes, it can be merged but please create a follow-up taks in the test-infra repository and add label "area/ci"